front/components/ideaDetail/IdeaCard.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <v-row
+    ref="page"
+    align="stretch"
+    :class="{ 'fixed-height': !isExpanded }"
+    class="elevation-2 ma-1 card"
+  >
+    <div class="tap-bar" @click="expandToggle"></div>
+    <slot></slot>
+  </v-row>
+</template>
+
+<script>
+export default {
+  name: 'IdeaCard',
+  props: {
+    expanded: Boolean
+  },
+  data() {
+    return {
+      isExpanded: this.expanded === undefined ? false : this.expanded
+    }
+  },
+  mounted() {
+    // Disable vertical scroll if card is not expanded
+    this.$refs.page.addEventListener(
+      'touchmove',
+      this.preventScrollOnMobileWhenCardIsNotExpanded
+    )
+  },
+  methods: {
+    preventScrollOnMobileWhenCardIsNotExpanded(event) {
+      if (this.isExpanded === false) {
+        event.preventDefault()
+      } else {
+        return true
+      }
+    },
+    expandToggle() {
+      this.isExpanded = !this.isExpanded
+      this.$emit('expand-toggle')
```

Comment:
where its used dont see

Reply:
Is you mean the `expand-toggle` event it's being used [here](https://github.com/ezl/dailyidea.com/blob/38a79b9cf37e307f714b6f82605050915d3882ab/front/pages/i/_shortId/_slug.vue#L22)